src/index.js

```diff
@@ -1 +1,85 @@
+/* eslint-disable no-console */
 'use strict';
+
+const http = require('http');
+const fs = require('fs');
+const zlib = require('zlib');
+const formidable = require('formidable');
+const { pipeline } = require('stream');
+
+const PORT = process.env.PORT || 3000;
+
+const server = new http.Server();
+
+server.on('request', (req, res) => {
+  const url = new URL(req.url, `http://${req.headers.host}`);
+  const pathname = url.pathname.slice(1) || 'index.html';
+
+  if (pathname === 'index.html') {
+    res.setHeader('Content-Type', 'text/html');
+
+    const file = fs.createReadStream('src/index.html');
+
+    file.pipe(res);
+
+    file.on('error', () => {
+      res.statusCode = 404;
+      res.end();
+    });
+  } else if (pathname === 'dowload') {
```

Comment:
```suggestion
  } else if (pathname === 'download') {
```

Reply:
Yeap, let's fix this and in the HTML file